components/tools/OmeroWeb/test/integration/test_api_projects.py

```diff
@@ -561,11 +566,13 @@ def test_project_update(self, user1):
         # Add @type and try again
         payload['@type'] = project_json['@type']
         rsp = _csrf_put_json(django_client, save_url, payload)
-        assert rsp['@id'] == project.id.val
-        assert rsp['Name'] == 'updated name'
-        assert 'Description' not in rsp
+        project_json = rsp['data']
+        assert project_json['@id'] == project.id.val
+        assert project_json['Name'] == 'updated name'
+        assert 'Description' not in project_json
         # Get project again to check update
-        pr_json = _get_response_json(django_client, project_url, {})
+        rsp = _get_response_json(django_client, project_url, {})
+        pr_json = rsp['data']
```

Comment:
maybe add another test starting with no description,  insert description and check if it is correctly updated